apps/remix-ide-e2e/src/tests/solidityUnittests.spec.ts

```diff
@@ -150,14 +150,16 @@ module.exports = {
       .clickLaunchIcon('solidityUnitTesting')
       .waitForElementPresent('*[data-id="uiPathInput"]', 3000)
       .clearValue('*[data-id="uiPathInput"]')
-      .setValue('*[data-id="uiPathInput"]', 'tests1')
+      .setValue('*[data-id="uiPathInput"]', 'tests')
```

Comment:
Why did this value change `tests1` to `tests`? Shouldn't this affect e2e and fail?